source/ways-of-working/common-pipeline/common-pipeline.html.md.erb

```diff
@@ -102,3 +102,14 @@ Within AKS, your application pods will exist within a namespace. The namespace w
 ### Troubleshooting build issues
 
 See [troubleshooting issues](../troubleshooting/). 
+
+### Tips and Tricks
+
+#### How to get to the Test URL
+
+1. Click 'Show environments'
+2. Click 'View deployment'
+
+e.g. [hmcts/pip-frontend#79:](https://github.com/hmcts/pip-frontend/pull/79)
```

Comment:
have they changed this? for me if I click view deployment it takes me straight through?